dev/com.ibm.websphere.appserver.javaee-8.0/com.ibm.websphere.appserver.javaee-7.0.feature

```diff
@@ -0,0 +1,28 @@
+-include= ~../cnf/resources/bnd/feature.props
```

Comment:
Isn't the name of this file incorrect?  It should be ...javaee-8.0.feature, correct?